docs/sphinx/reference-sources.rst

```diff
@@ -455,7 +455,15 @@ Source Definition Structure (obs_source_info)
    - **OBS_ICON_TYPE_TEXT**            - Text
    - **OBS_ICON_TYPE_MEDIA**           - Media
    - **OBS_ICON_TYPE_BROWSER**         - Browser
-   - **OBS_ICON_TYPE_CUSTOM**          - Custom (not implemented yet)
+   - **OBS_ICON_TYPE_CUSTOM**          - Custom
+
+.. member:: const char *(*obs_source_info.get_dark_icon)(void *type_data)
+
+   Set icon used for dark themes. Make sure icon_type is set to OBS_ICON_TYPE_CUSTOM.
+
+.. member:: const char *(*obs_source_info.get_light_icon)(void *type_data)
+
+   Set icon used for light themes. Make sure icon_type is set to OBS_ICON_TYPE_CUSTOM.
```

Comment:
Why do these entries say "Set icon" when the functions are named "get" and the inline comment in the header says "Gets"?

Reply:
Updated to use get.